Wrappers.h

```diff
@@ -1,4 +1,9 @@
 #pragma once
+#include "Computer.h"
+#include "Car.h"
+#include "Synth.h"
+#include "Musician.h"
+#include "Garage.h"
```

Comment:
Don't include what you don't use
Forward declare types that are not used in header files.
Include types that are used in CPP files
- [ ] revise